defaults/main.yml

```diff
@@ -41,6 +41,7 @@ telegraf__influxdata__dependent_packages:
 # FIXME FIXME FIXME FIXME FIXME FIXME
 # Opisać poniższe zmienne
 
+telegraf__influxdb_enable: True
```

Comment:
Może dla konsekwencji tutaj też dodać `output` w nazwie?

Reply:
poniżej są inne zmienne konfiguracyjne z telegraf__influxdb_*,  ja bym zostawił tak jak jest, z komentarzem że dotyczy to wysyłki do Influksa